azavea/pfb-network-connectivity
# Add hand-defined region lists for Denmark and Norway

## Overview

Since `pycountry` doesn't have the desired entities, this adds hand-coded subdivisions lists for each, using the names and abbreviations sent by PFB.

### Demo

![image](https://user-images.githubusercontent.com/6598836/56759998-37af3600-6768-11e9-88a4-aa5fa8bec04a.png)

![image](https://user-images.githubusercontent.com/6598836/56760010-3d0c8080-6768-11e9-9d1a-ffc0009affe8.png)

## Testing Instructions

- Make a neighborhood.  Confirm that the "State/Province" droplist now shows up for Denmark and Norway, and contains the list of regions.
- Import an analysis into the neighborhood you made, and confirm that it shows up on the site and the filtering droplists work correctly.

## Checklist

~- [ ] Add entry to CHANGELOG.md~ (covered by an existing entry)
